update_substrate_template.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# This is an automation for updating Substrate Node Template
+# https://github.com/substrate-developer-hub/substrate-node-template
+# Triggered on push of new branch on polkadot release (i.e polkadot-v0.9.26) on substrate repo
+#
+# TODO: If any step fails => send a notification to some channel
+#
+# Usage:
+# Update Cumulus -> Substrate Parachain Template
+# ./pipeline-scripts/update_substrate_template.sh
+#        --repo-name substrate-parachain-template
+#        --template-path "parachain-template"
+#        --github-api-token "$GITHUB_TOKEN"
+#        --polkadot-branch "$CI_COMMIT_REF_NAME"
+#
+# Update Substrate -> Substrate Node Template
+# ./pipeline-scripts/update_substrate_template.sh
+#        --repo-name substrate-node-template
+#        --template-path "bin/node-template"
+#        --github-api-token "$GITHUB_TOKEN"
+#        --polkadot-branch "$CI_COMMIT_REF_NAME"
+
+set -eu -o pipefail
+
+. "$(dirname "${BASH_SOURCE[0]}")/utils.sh"
+
+# substrate-parachain-template or substrate-node-template
+get_arg required --repo-name "$@"
+target_repo_name="$out"
+
+# "bin/node-template" or "parachain-template"
+get_arg required --template-path "$@"
+template_path="$out"
+
+# GITHUB_TOKEN or other
+get_arg required --github-api-token "$@"
+github_api_token="$out"
+
+# $CI_COMMIT_REF_NAME
+get_arg required --polkadot-branch "$@"
+polkadot_branch="$out"
+
+target_org="substrate-developer-hub"
+timestamp=`date +%Y-%m-%d_%H-%M-%S`
```

Comment:
```suggestion
timestamp=$(date +%Y-%m-%d_%H-%M-%S)
```
It's more preferred to use $(...) than ticks